pmd-core/src/main/java/net/sourceforge/pmd/lang/AbstractLanguageVersionHandler.java

```diff
@@ -80,4 +81,9 @@ public DFAGraphRule getDFAGraphRule() {
     public LanguageMetricsProvider<?, ?> getLanguageMetricsProvider() {
         return null;
     }
+
+    @Override
+    public DesignerBindings getDesignerBindings() {
```

Comment:
Do we need to add the `@Experimental` annotation here as well? I know, it's on the interface, but....
Ok, we don't have it for the getLanguageMetricsProvider either

Reply:
Hm I think the interface doc should be sufficient here, plus `DesignerBindings` is experimental itself